cmd/util.go

```diff
@@ -602,6 +602,7 @@ func addCIMetadataToEnvironment(env map[string]string) {
 	}
 	env[backend.CISystem] = string(vars.Name)
 	addIfSet(backend.CIBuildID, vars.BuildID)
+	addIfSet(backend.CIBuildNumer, vars.BuildNumber)
```

Comment:
Is this what you are talking about, @chrsmith ?

Reply:
Yes. Totally missed it. All is well 👍

Reply:
Yes. Totally missed it. All is well 👍